libs/shared/connected/src/lib/fiskistofa/SidebarShipSearchInput/SidebarShipSearchInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import { AsyncSearchInput, Box, Text } from '@island.is/island-ui/core'
+
+interface SidebarShipSearchInputProps {
+  shipDetailsHref?: string
+  shipSearchHref?: string
+  placeholder?: string
+  label?: string
+}
+
+export const SidebarShipSearchInput = ({
+  shipDetailsHref = '/s/fiskistofa/skip',
+  shipSearchHref = '/s/fiskistofa/skipaleit',
+  placeholder = 'Skipaskrárnúmer eða nafn',
+  label = 'Skoða skip',
```

Comment:
i18n missing.

Reply:
I'm using the connected component content type in Contentful and the translations are passed throught as props via the JSON field in Contentful.